src/app/private/[uid]/admin/layout.tsx

```diff
@@ -1,12 +1,42 @@
+"use client";
+
+import Sidebar, { ISideBar } from "@components/Sidebar";
+import { faHome, faHouseLock, faUser } from "@fortawesome/free-solid-svg-icons";
+import React, { useContext } from "react";
+import { UserContext } from "src/context/UserProvider";
+
 interface IAdminLayout {
   children: React.ReactNode;
 }
 
-// @TODO - Add navigation bar
 const AdminLayout = ({ children }: IAdminLayout) => {
+  const { user } = useContext(UserContext);
+  const buttons: ISideBar["buttons"] = React.useMemo(
```

Comment:
Layout files for `Admin`, `Chapter Leader`, and `User` can use the `Sidebar` to render what's on screen.